payload/Library/installapplications/installapplications.py

```diff
@@ -703,6 +733,9 @@ def main():
                     iaslog("Waiting for user script to complete: %s" % path)
                     time.sleep(0.5)
 
+                # Log userscript item rumtime
+                write_item_total_runtime(stage, name, userscript_runtime_start)
+
```

Comment:
You could likely also reduce the lines of repeated code by writing the runtime at the very bottom of the `for` loop. It should honestly just be called once.

Reply:
Same as above. I will make another commit, and update the PR. Thanks @erikng!